api/cpp/include/slint-platform.h

```diff
@@ -93,6 +103,9 @@ class WindowAdapter
 
     /// Re-implement this function to provide a reference to the renderer for use with the window
     /// adapter.
+    ///
+    /// Your re-implementation should contain a renderer such as SoftwareRenderer or SkiaRenderer
+    /// and you can return a reference to it
```

Comment:
Changing to "must" because this function is required after all. But feel free to stay with "can" if you prefer.

```suggestion
    /// and you must return a reference to it.
```